CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 1.3.0
+
+- End empty doc comment with full stop ([#202])
```

Comment:
```suggestion
- End `empty` doc comment with full stop ([#202])
```
To make it clear that we're talking about the function, not some "empty doc comment" (whatever that is, but I was confused at first).